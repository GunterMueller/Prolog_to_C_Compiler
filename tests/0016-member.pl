main :-
	member(X, [1,2,3]), display(X), nl, fail.
main :-
	member(2, [1,2,3]), display('yes'), nl,
	(member(3, [1,2]); display('no'), nl),
	member(1, [Y|_]), display(Y), nl.
