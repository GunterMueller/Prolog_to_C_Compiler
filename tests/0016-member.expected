1
2
3
yes
no
1
